src/components/DeleteHouse/DeleteHouse.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { IoCaretBackOutline } from 'react-icons/io5';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { houseData } from '../../mockData';
+import styles from './DeleteHouse.scss';
+import { deleteHouse } from '../../Redux/Reducers/deleteHouseSlice';
+
+const DeleteHouse = () => {
+  const dispatch = useDispatch();
+  const [formData, setFormData] = useState({
+    name: '',
+    city: '',
+    image: '',
+    appartment_fee: '',
+    description: '',
+  });
+
+  const handleDelete = (e) => {
+    e.preventDefault();
+    dispatch(deleteHouse(formData));
+    setFormData({
+      name: '',
+      city: '',
+      image: '',
+      appartment_fee: '',
+      description: '',
+    });
+  };
```

Comment:
This is not how to dispatch a delete action. The endpoint does not need an data from the form. The action must be dispatched when the user clicks the delete button for a specific house using it's id.

Reply:
Ok thanks